examples/kubernetes/clustertemplate/base-clustertemplates.tf

```diff
@@ -0,0 +1 @@
+../clustertemplates/main.tf
```

Comment:
do we need this file now?

Reply:
no i removed it everywhere as we do not link template as selection from templates